dali/operators/generic/squeeze.cc

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include <vector>
+
+#include "dali/core/math_util.h"
+#include "dali/core/static_switch.h"
+#include "dali/core/tensor_layout.h"
+#include "dali/core/tensor_shape_print.h"
+#include "dali/operators/generic/squeeze.h"
+#include "dali/pipeline/data/views.h"
+
+namespace dali {
+
+DALI_SCHEMA(Squeeze)
+  .DocStr(R"code(Collapses the dimensions given as axes or axis_names.
+
+It's an error to collapse dim that would cause the volume to change
+(we can collapse a non-unit dim if a non-collapsed dim is 0).)code")
+  .NumInput(1, 2)
```

Comment:
```suggestion
  .NumInput(1)
```
Squeeze takes only a single input